app/Http/Controllers/Api/UsersController.php

```diff
@@ -53,10 +53,38 @@ public function __construct(Registrar $registrar, Manager $manager, UserReposito
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(UserRequest $request)
     {
-        // @TODO: temp for now, likely want to use repository and transform.
-        return User::all();
+        $data = $request->all();
```

Comment:
Not sure if you really need to separate this into a `$data` variable... it's only really used below for finding the user, but then you switch to `$request['param_name']`. Would recommend just sticking w/ `$request['...']` approach throughout or `$request->input('...')`, etc :)